### PR TITLE
added missing use statement to fix #8572

### LINKED ISSUE #8572: Strange issue with TimestampBehavior
UpdatedAt works well after any moficiation in row. But, I can't say the same for row creation. 
After new row creation `createdAt` still 0000-...
      
    public function behaviors()
      {
        return [
            [
                'class' => TimestampBehavior::className(),
                'createdAtAttribute' => 'createdAt',
                'updatedAtAttribute' => 'updatedAt',
                'value' => new Expression('NOW()')
            ],
          ];
      }

Here is database table: http://prntscr.com/79xt5n

And here is model: https://gist.github.com/turalsprofile/85eca5d307074b42a215

What am I doing wrong?

### PR DESCRIPTION
**Pull request by cebe**
- added unit tests for timestamp behavior and insert with expression
- fixed issue #8572 

tests may not pass on postgres, waiting for travis so I can fix it.

problem has been introduced by 6e48ed0 (#8034, 2.0.5) so no changelog is needed.

/cc @nineinchnick

### CODE REVIEW

**Comment by cebe** [comment]
memo to myself:

- [ ] add tests for batchInsert too.

File: framework/db/QueryBuilder.php [hunk 1]
```diff
@@ -9,6 +9,7 @@
 
 use yii\base\InvalidParamException;
 use yii\base\NotSupportedException;
+use yii\db\Expression;
```
Comment (klimov-paul) [inline framework/db/QueryBuilder.php]: Why?
`yii\db\QueryBuilder` and `yii\db\Expression` are under the same namespace.

Comment (cebe) [inline framework/db/QueryBuilder.php]: good point, for some reson my PHPStorm complained here...

Comment (cebe) [inline framework/db/QueryBuilder.php]: fixed and merged. aa5887f

**Pull request closed without merging**